src/screens/OtherProfile/index.js

```diff
@@ -561,15 +427,12 @@ const OtherProfile = () => {
           isAnonimity={isAnonimity}
           bio={dataMain.bio}
           openBio={openBio}
+          dataMain={dataMain}
           isSignedMessageEnabled={isSignedMessageEnabled}
           showSignedMessageDisableToast={showSignedMessageDisableToast}
           loadingGenerateAnon={loadingGenerateAnon}
           avatarUrl={profile.myProfile.profile_pic_path}
           anonProfile={anonProfile}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The props passed to `BioAndChat` component have been modified. The `onSendDM`, `setDMChat`, `loadingSendDM`, and `dmChat` props have been removed. Make sure these changes do not affect the functionality of the `BioAndChat` component.
```diff
- onSendDM={onSendDM}
- setDMChat={setDMChat}
- loadingSendDM={loadingSendDM}
- dmChat={dmChat}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->